packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -2069,12 +2069,12 @@ export type MicrophoneButtonSelector = (state: CallClientState, props: CallingBa
 
 // @public
 export interface MicrophoneButtonStrings {
-    micPrimaryActionSplitButtonTitle?: string;
```

Comment:
intentional breaking change - this is a new string to 1.6.0

Reply:
Are ARB adamant we make this change? How are we planning on communicating this change to our customers?

Reply:
Sorry, I poorly phrased that - the breaking change is intentional to this PR. This is not a breaking change to customers as this did not exist in 1.5.0, this is a new string in 1.6.0